README.md

```diff
@@ -49,15 +49,16 @@ Collection of awesome [wagmi](https://github.com/wagmi-dev/wagmi)-related projec
 
 ### Templates
 
-- [create-wagmi](https://github.com/wagmi-dev/create-wagmi) — Create wagmi apps with one command
-- [create-rainbowkit](https://github.com/rainbow-me/rainbowkit/tree/main/packages/create-rainbowkit) — Scaffold a new RainbowKit + wagmi + Next.js app
-- [web3-scaffold](https://github.com/holic/web3-scaffold) — Quickly get up and running on web3
 - [create-web3](https://www.npmjs.com/package/create-web3) — Run `npx create-web3` in your terminal for a quick starter kit
+- [create-web3-frontend](https://github.com/dhaiwat10/create-web3-frontend)
+- [create-rainbowkit](https://github.com/rainbow-me/rainbowkit/tree/main/packages/create-rainbowkit) — Scaffold a new RainbowKit + wagmi + Next.js app
+- [create-wagmi](https://github.com/wagmi-dev/create-wagmi) — Create wagmi apps with one command
+- [Nexth](https://github.com/wslyvh/nexth/) — a Next.js + Ethereum starter kit to quickly ship Web3 Apps ⚡
```

Comment:
Let's just add this in and keep everything the same as it was before.

Reply:
The contribution guidelines (in OP) actually state to alphabetize the list. I figured to update/correct that.
Happy to revert, but you probably want to update the guidelines in that case.

Reply:
Exactly! Thanks for making it fit those guides. I took a quick look earlier when I wrote the comment and saw that some still need alphabetizing. Once that is done, lgtm!